components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/internal/SAMLApplicationMgtListener.java

```diff
@@ -80,6 +80,11 @@ public boolean doPreDeleteApplication(String applicationName,
                 .getApplicationExcludingFileBasedSPs(applicationName, tenantDomain);
 
         if (sp != null) {
+            // TODO remove after testing
+            if (log.isDebugEnabled()) {
+                log.debug("Service provider associated with the application name: " + applicationName +
+                        " is not null");
```

Comment:
Better to add a method functionality specific message here.

Reply:
Fixed